src/components/placeholder.js

```diff
@@ -37,7 +48,8 @@ class Placeholder extends React.Component {
     return (
       props.children != this.props.children ||
       props.className != this.props.className ||
-      props.parent != this.props.parent ||
+      props.firstOnly != this.props.firstOnly ||
+      (props.firstOnly && (props.parent != this.props.parent)) ||
```

Comment:
This is because `parent` prop is used in `Placeholder` only when `firstOnly` is set to true. So this would prevent a needless re-render (super edgecase where someone has `firstOnly` set to false and is passing parent accidentally and that changes).